sound/soc/intel/common/soc-acpi-intel-ehl-match.c

```diff
@@ -8,8 +8,29 @@
 
 #include <sound/soc-acpi.h>
 #include <sound/soc-acpi-intel-match.h>
+#include "../skylake/skl.h"
+
+static struct skl_machine_pdata ehl_pdata = {
+	.use_tplg_pcm = true,
+};
```

Comment:
not needed, remove all dependencies on SKL stuff.

Reply:
OK